.stickler.yml

```diff
@@ -0,0 +1,3 @@
+linters:
```

Comment:
- [ ] Kindly enable the stickler for this repo. You will have to go to https://stickler-ci.com/ to do so.